README.md

```diff
@@ -1,2 +1,159 @@
-# historian
-A solution for extracting historical information from a Metasys® Server
+# Historical Data Extractor
+
+The Historical Data Extractor is a series of apps that extract historical data from the *Metasys* system into a data store of your choice and in a format that can be easily read by a BI tool such as Power BI or Tableau.
+
+There are three apps included in this source code:
+
+- [Quick Extract](#quick-extract)
+- [Discovery](#discovery)
+- [Extraction](#bulk-extraction)
+
+There is also a mechanism in place for you to be able to add your own data store for the historical data. You will simply need to implement one interface to do so. Instructions on that can be found in the [Data Storage](#data-storage) section.
+
+## Prerequisites
+
+The three apps of the Historical Data Extractor require the following prerequisites prior to running the apps:
+
+- Download and install dotnet core (https://www.microsoft.com/net/learn/get-started/windows) on the client computer that you run the app from.
+- Clone this repository.
+- Credentials (username and password) to the *Metasys* system. We recommend a user with view-only permissions.
+- Credentials to the SQL database (for the dbconnection string for #discovery and #extraction).
+
+## Quick Extract
+
+To run the Quick Extract app, run the command below in from the `Quick Extract` folder. The results for the last day are stored in the current directory in a new file named timeseries.csv.
```

Comment:
Might want to be more explicit about the name of the folder. I don't think it's Quick Extract, it's HistoricalDataFetcher.QuickExtract. Why not even link to the actual folder in the rep?